frontend/packages/volto-intranet/package.json

```diff
@@ -53,6 +53,7 @@
   ],
   "dependencies": {
     "@kitconcept/core": "workspace:*",
+    "react-aria-components": "^1.6.0",
```

Comment:
@sneridagh Do we need a different version by now to be compatible with what you added in Volto core?

Reply:
@iFlameing Regarding `@plone/components`, we should use the components in there (CSS ones), and customize them when needed (which is the case). It seems that you want a TextField, but with different CSS, right? Then pass your own CSS and the others will be overridden. Then you don't need to rely on `react-aria-components`. Could be that you want anyways use the vanilla components from RAC, that is fine too. But in that case, we have to sync with the one used in `@plone/components`. This has to be done manually, but I am working on the catalog pnpm feature, so we will force the same version everywhere we need. Until this is not in place if you _really_ need vanilla RAC, you have to do it manually.